docs/Documentation/publish-via-cli.md

```diff
@@ -0,0 +1,27 @@
+# Publish from Command Line 
+
+![Publish Via CLI](../../assets/images/publish-via-cli.png)
+
+1. Install the @stoplight/cli package from NPM 
+
+```
+Npm i -g @stoplight/cli
+
+# The “stoplight” cli should now be available globally 
+
+Stoplight help 
```

Comment:
Lowercase `stoplight` and it is `-h`, e.g. `stoplight -h`. If it says `stoplight help` in the instructions in app we need to fix that.